src/plugins/usrbg/index.css

```
@@ -0,0 +1,32 @@
+[class*="userProfile"] [class*="bannerPremium"],
```

you can make this more elegant using the :is operator
```suggestion
:is([class*="userProfile"], [class*="userPopout"]) [class*="bannerPremium"],
```